plugins/foreman_statistics/1.0/index.md

```diff
@@ -6,17 +6,18 @@ version: 1.0
 #robots: noindex
 ---
 
-<div class="bs-callout-warning bs-callout">
-  <h4>This is a Work in progress</h4>
-  Plugin will be available only in the next major release!
-</div>
-
 # 1. {{ page.title }}
 
 This plugin enables Foreman to show a nice statistics dashboard and create Trend charts.
 
+## 1.1 Release notes
+
+### Drop the cron job
+
+Foreman statistics moves trend calculation from a cronjob to a Dynflow job and no manual steps are required.
+Users upgrading from older versions should ensure to remove any cronjobs that call `foreman-rake trends:counter`.
```

Comment:
Older versions may imply older versions of the plugin. May be call out that it was in core prior to Foreman 2.3?

Reply:
reworded to `from Foreman 2.2 and older`